src/Npgsql/BackendMessages/CommandCompleteMessage.cs

```diff
@@ -32,8 +32,8 @@ namespace Npgsql.BackendMessages
     class CommandCompleteMessage : IBackendMessage
     {
         internal StatementType StatementType { get; private set; }
-        internal uint OID { get; private set; }
-        internal uint Rows { get; private set; }
+        internal ulong OID { get; private set; }
```

Comment:
This doesn't seem right, IIRC `OID` is always a uint.

Reply:
Problem comes from below `static ulong ParseNumber(byte[] bytes, ref int pos)`

Reply:
Can we find documentation that `OID` is `uint`, and then just apply the explicit cast?

Reply:
Per [the docs](https://www.postgresql.org/docs/current/static/datatype-oid.html):

> The `oid` type is currently implemented as an unsigned four-byte integer.

Reply:
@austindrenski done